primitives/direct_rpc_server/Cargo.toml

```diff
@@ -0,0 +1,49 @@
+[package]
+name = "direct-rpc-server"
```

Comment:
We'll probably want a prefix in the name here?

Reply:
Yes for now I prepended `substratee` everywhere to easily identify where we need to put a proper prefix.